src/useRouteQueryParam/README.md

```diff
@@ -0,0 +1,18 @@
+# useRouteQueryParam
+The `useRouteQueryParam` gives you a reactive value for a query param. So you can bind directly to `foo` in the route `example.com?foo=bar`
+
+## Example
+```typescript
+import { useRouteQueryParam } from '@prefecthq/vue-compositions'
+
+const foo = useRouteQueryParam('foo')
```

Comment:
might be worth showing that you can also write to `foo`, which will push to router

Reply:
updated